scss/inc/_navigator.scss

```diff
@@ -422,11 +422,9 @@ $borderColor: $textHighlight;
     }
     .qti-navigator-item {
         background: white();
-        &.active {
-            background: lighten($lightBlueGrey, 5);
-        }
-        &:hover {
-            background: lighten($lightBlueGrey, 3);
+        &.active, &:hover {
```

Comment:
Those states had different colors before.
Do we expect to use the same for both?

Reply:
As I understand yes, because with previous colors on hover, the issue remains, we can't see properly the text. I will double-check

Reply:
As I talk, customer (and probably TAO) only want to change one, but we have an issue when you hover an active element. So, with Luc and Kristina we are moving this for see what customer wants on hover, and what we want on product.

https://files.slack.com/files-pri/T04V23RQT-F018TEFPZ6X/image.png